backend/services/captures.py

```
@@ -0,0 +1,219 @@
+    # canonical WAV we hand to whisper.
+    try:
+        audio, sr = load_audio(str(raw_path))
+        duration_ms = int((len(audio) / sr) * 1000) if sr else None
+    except Exception as decode_err:
+        logger.warning(
+            "Could not decode capture %s (%s): %r", capture_id, suffix, decode_err
+        )
+        audio, sr = None, None
+        duration_ms = None
+
+    _WHISPER_NATIVE_FORMATS = (".wav", ".mp3", ".flac", ".ogg")
```

Ruff (0.15.11)

[warning] Variable `_WHISPER_NATIVE_FORMATS` in function should be lowercase

(N806)